app/views/filters/_autocomplete.html.erb

```diff
@@ -12,7 +12,6 @@
                               # of the #connect lifecycle hook of the filter--filters-form
                               # Stimulus controller.
                             }.merge(autocomplete_options.except(:component)),
-                            class: 'form--field',
```

Comment:
This class seems to break the alignment of the autocomplete input field. It happens on the project page and the meeting filter page too. Removing it fixes the alignment.
**Before:**
<img width="1041" alt="image" src="https://github.com/user-attachments/assets/0f2955d7-f7f2-46e4-b658-cf0b595cb2c4" />
<img width="1044" alt="image" src="https://github.com/user-attachments/assets/543b94cd-b169-40df-b2ea-4c093a14dd05" />


**After:**
<img width="1049" alt="image" src="https://github.com/user-attachments/assets/9d69d690-5f73-400d-b226-e4e2895212f4" />
<img width="1054" alt="image" src="https://github.com/user-attachments/assets/e090f6a8-53ea-43fc-9d87-436ea48a98ce" />